.github/workflows/frontend-ci.yml

```
@@ -0,0 +1,25 @@
+name: Frontend CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
```

```suggestion
    steps:
    - uses: actions/checkout@v4
    - uses: actions/setup-node@v4
      with:
        node-version: 20
        cache: 'npm'
```


(Apologies for the fly-by review, great PR ! )

This suggestion is to target node 20 (https://github.com/XavB64/lowtrip/blob/998ac704b96d8d4089b720628f57e093db3b918a/frontend/package.json#L13) and thus run CI only on the targeted version instead of multiple jobs including some EOL versions of node